shell/browser/mac/electron_application_delegate.mm

```diff
@@ -157,4 +158,44 @@ - (IBAction)newWindowForTab:(id)sender {
   electron::Browser::Get()->NewWindowForTab();
 }
 
+- (void)application:(NSApplication*)application
+    didRegisterForRemoteNotificationsWithDeviceToken:(NSData*)deviceToken {
+  // https://stackoverflow.com/a/16411517
+  const char* tokenData = static_cast<const char*>([deviceToken bytes]);
```

Comment:
code style for local variables is to use `snake_case`. it's a bit confusing when in Objective C land because argument names use `camelCase`! but the other functions in this file use `snake_case` for local variables so let's match suit.